### DISCUSSION TITLE
[PSD8] Esercizio Slide 1

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#160] [Esercizi (A-L)]
Minimizzare le equazioni dell'Esercizio 2.18 (#159 ) usando le mappe-K. Fornire sia la rappresentazione minima SOP, sia la POS

![image](https://user-images.githubusercontent.com/69389461/210059660-2033ee68-825c-4a30-add7-da1b290f0469.png)

**Comment by Elia-Belli** [comment]
- (a) $Y = \bar B + C$
- (b) $Y = B + \bar C D + AD = (B + D) (A + B + \bar C)$
- (c) $Y = \bar A B \bar C D + ABCD + B \bar D E = B (D+E) (A+\bar C + \bar D)(\bar A + C + \bar D)$

![211633060-7104efe6-4316-4c04-bf45-bf86946adf54](https://user-images.githubusercontent.com/107944476/213290459-34dfe15e-02fe-48ce-a39f-131fa016d201.png)
Nella (c) prendo un implicante primo da 16, uno da 8 e due da 4

**Comment by FeddyLix17** [comment]
tutto corretto tranne le forme pos di **(b)** e **(c)**

- (b) mi viene (A+B+C&#x0304;)(B+D)
- (c) mi viene (A&#x0304;+B&#x0304;+C&#x0304;+D&#x0304;)(A+B&#x0304;+C+D&#x0304;)(B&#x0304;+D+E&#x0304;)

**Comment by Elia-Belli** [comment]
Ho corretto la (b) e modificato la (c) ma viene diversa dalla tua

**Comment by Elia-Belli** [comment]
@Amirhossein-2020 Per essere precisi dice "K-maps work well for problems with up to four variables", ovvero è un metodo che funziona **bene** per funzioni fino a 4 variabili, ma non dice che non funziona altrimenti.

Se facciamo un pò di astrazione è possibile pensare di sovrappore due K-maps da 4 variabili l'una, in modo che le celle che differiscano solo per la nuova quinta variabile siano adiacenti.

**Comment by Amirhossein-2020** [comment]
> @Amirhossein-2020 Per essere precisi dice "K-maps work well for problems with up to four variables", ovvero è un metodo che funziona **bene** per funzioni fino a 4 variabili, ma non dice che non funziona altrimenti.
> 
> Se facciamo un pò di astrazione è possibile pensare di sovrappore due K-maps da 4 variabili l'una, in modo che le celle che differiscano solo per la nuova quinta variabile siano adiacenti.

Si, hai ragione, anch'io ho disegnato due K-maps per la domanda (c) ed ho trovato la risposta simile(solo che i miei K-maps sono diversi per A ed A' invece di E ed E', comunque la risposta è sempre uguale).
Anche affermo le sue risposte per (A) e (B).
Grazie

**Comment by Amirhossein-2020** [comment]
Adesso ho controllato che:
(C) POS mi viene: (B)(D+E)(A+C+D')(A'+C'+D')
magari io sbagli.

**Comment by Elia-Belli** [comment]
@Amirhossein-2020 avevo sbagliato l'ultimo literal, comunque mi viene diverso. Prova ad allegare la tua k-map

**Comment by Amirhossein-2020** [comment]
Ok.
Scusa che è scarabocchiato. 😅

![IMG_0796](https://user-images.githubusercontent.com/75175964/213499064-01f88a96-880f-42bd-9995-310af3f31d37.jpg)
![New Microsoft PowerPoint Presentation](https://user-images.githubusercontent.com/75175
![IMG_0797](https://user-images.githubusercontent.com/75175964/213499183-9dffc43c-b36d-4025-b0d2-2eb63dac70ae.jpg)
964/213499157-4e6c1f04-99f5-4586-9cab-7040118eb13d.jpg)
